sdks/v3-sdk/src/utils/swapMath.ts

```diff
@@ -26,12 +26,13 @@ export abstract class SwapMath {
       feeAmount: JSBI
     }> = {}
 
+    feePips = JSBI.BigInt(feePips)
```

Comment:
Does `JSBI.BigInt` work with JSBI input or should we do this conditionally based on the type?

Reply:
it will take a JSBI input (or else the tests would be failing actually)